statistik/controller.py

```diff
@@ -67,10 +68,19 @@ def get_avg_ratings(chart_ids, game=IIDX, user_id=None, include_reviews=False):
         if specific_reviews:
             # for each rating type, average the scores in matched reviews
             for rating_type in SCORE_CATEGORY_NAMES[game]:
-                avg_rating = "%.1f" % round(statistics.mean(
-                    [getattr(review, rating_type)
-                     for review in specific_reviews
-                     if getattr(review, rating_type) is not None] or [0]), 1)
+                # Attempt to ignore outlier reviews by calculating the average, then removing reviews
+                # with a score beyond some threshold away from that average
+                rating_reviews = [getattr(review, rating_type)
+                                  for review in specific_reviews
+                                  if getattr(review, rating_type) is not None] or [0]
+                # If there are fewer than three reviews, don't bother eliminating outliers
+                if len(rating_reviews) < 3:
+                    avg_rating = "%.1f" % round(statistics.mean(rating_reviews), 1)
```

Comment:
Don't do formatting until right when you set `ret[chart][rating_type]`, it's cleaner and makes your check below for 0 easier